spec/models/review_spec.rb

```diff
@@ -8,11 +8,14 @@
   it { should respond_to :rating }
   it { should validate_presence_of :content }
   it { should validate_presence_of :rating }
+  it { should validate_presence_of :video }
 
-  context 'for rating values between 0 and 5' do
+  let(:video) { video = Fabricate(:video) }
+
+  context 'for rating values between 1 and 5' do
```

Comment:
```ruby
  it { should validate_inclusion_of(:rating).in_array((1..5).to_a) }
```
not working. Method validate_inclusion_of undefined. Error:

NoMethodError: undefined method `validate_inclusion_of' for #<RSpec::Core::ExampleGroup::Nested_8:0x00000007c0a588>
./spec/models/review_spec.rb:12:in `block (2 levels) in <top (required)>'